src/VisualStudio/Core/Def/xlf/ServicesVSResources.zh-Hant.xlf

```diff
@@ -72,6 +102,26 @@
         <target state="new">Kind</target>
         <note />
       </trans-unit>
+      <trans-unit id="Make_abstract">
+        <source>Make abstract</source>
+        <target state="new">Make abstract</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Members">
+        <source>Members</source>
+        <target state="new">Members</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Namespace_0">
+        <source>Namespace: {0}</source>
+        <target state="new">Namespace: {0}</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Namespace_global">
+        <source>Namespace: global</source>
+        <target state="new">Namespace: global</target>
```

Comment:
what are these for?

Reply:
This is a content of a tool tip. When user's mouse select on the destination, the tool tip will tell the the user its namespace. So it one is used to when there is no namespace declaration.